### PR TITLE
Mongodb

### PR DESCRIPTION
**Pull request by chris-yooo**
:) mongo GET and POST method implemented

### CODE REVIEW

File: api/messages/index.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+              return {_id: message._id, msg: message.msg};
+            })
+          );
+        } else {
+          const error = new Error({message: 'Message not known', status: 500});
+          throw error;
+        }
+      } catch (err) {
+        console.log(err);
+        return response.status(500).json({message: 'Server error'});
```
Comment (PReichetanz) [inline api/messages/index.js:22]: This is more or less the same, isn't it? Is there a "better" error you can throw in line 17-18?
I think you might rather want to say something like "sorry, there are no messages yet" or similar if `messages` is not truthy.

File: api/messages/index.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+//       return response.json(message);
+//     } else {
+//       const error = new Error({message: 'Message not known', status: 500});
+//       throw error;
+//     }
+//   } catch (err) {
+//     console.log(err);
+//     return response.status(500).json({message: 'Server error'});
+//   }
+// };
```
Comment (PReichetanz) [inline api/messages/index.js:77]: Please delete unused code before you merge into main.

File: src/components/Chat/Chat.jsx [hunk 3]
```diff
@@ -1,7 +1,10 @@
 export default function Chat({messages, goToScrollDown}) {
-  return messages.map(message => {
+  console.log(messages);
+
+  return messages?.map(message => {
+    // console.log(message);
```
Comment (PReichetanz) [inline src/components/Chat/Chat.jsx:5]:
```suggestion
```

File: src/components/Chat/Chat.jsx [hunk 4]
```diff
@@ -1,7 +1,10 @@
 export default function Chat({messages, goToScrollDown}) {
-  return messages.map(message => {
+  console.log(messages);
```
Comment (PReichetanz) [inline src/components/Chat/Chat.jsx:2]:
```suggestion
```

**Review by PReichetanz — COMMENTED** [review]
Nice work, just a few issues to consider 🚀

File: api/messages/index.js (hunk 2 above)
Comment (chris-yooo) [reply api/messages/index.js:77]: ok

File: api/messages/index.js (hunk 1 above)
Comment (chris-yooo) [reply api/messages/index.js:22]: good idea

**Pull request merged**